GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -109,6 +114,9 @@ private abstract class Tree
             private readonly IGitUICommandsSource _uiCommandsSource;
 
             private readonly CancellationTokenSequence _reloadCancellationTokenSequence = new CancellationTokenSequence();
+            protected CancellationToken CurrentReloadCancellationToken;
```

Comment:
We are using a very leaky abstraction if we have to resort to exposing fields.
If it written to outside the class? If not, please turn this into a RO property

Reply:
You're right. I got rid of this member, and am now passing down the token to FillTreeViewNode, etc.